### PR TITLE
Swagger Auth

### PR DESCRIPTION
**Pull request by smizell**
Initial Swagger auth PR.

### CODE REVIEW

**Comment by smizell** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;fqPmvlQK&#x2F;26-swagger-zoo-examples"> Swagger Zoo examples</a>

**Comment by smizell** [comment]
cc @pksunkara

File: fixtures/features/swagger/auth-basic.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  /with-security:
+    get:
+      security:
+        - customBasic: []
+      responses:
+        200:
+          description: With Security
+          schema:
+            type: string
+  /no-security:
```
Comment (pksunkara) [inline fixtures/features/swagger/auth-basic.yaml]: Why is this resource needed?

File: fixtures/features/swagger/auth-token-in-header.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  /with-security:
+    get:
+      security:
+        - customApiKey: []
+      responses:
+        200:
+          description: With Security
+          schema:
+            type: string
+  /no-security:
```
Comment (pksunkara) [inline fixtures/features/swagger/auth-token-in-header.yaml]: not needed.

File: fixtures/features/swagger/auth-oauth2-implicit.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    flow: implicit
+    authorizationUrl: "http://example.com/oauth/authorize"
+    scopes:
+      scope1: Test Scope 1
+      scope2: Test Scope 2
+paths:
+  /with-security:
+    get:
+      security:
+        - customOauth2: []
```
Comment (pksunkara) [inline fixtures/features/swagger/auth-oauth2-implicit.yaml]: You want to do no scopes here?

**Comment by pksunkara** [comment]
Please remove `/no-security` when paired with `/with-security`. Unnecessary noise in the fixtures.

**Comment by smizell** [comment]
@pksunkara Addressed comments.

**Comment by pksunkara** [comment]
@smizell Generated Refract.

**Pull request merged**